Applications/SlicerApp/Testing/Python/DICOMReaders.py

```diff
@@ -97,23 +97,27 @@ def test_Part1DICOM(self):
         "fileName": "Mouse-MR-example-where-GDCM_fails.zip",
         "name": "Mouse-MR-example-where-GDCM_fails",
         "seriesUID": "1.3.6.1.4.1.9590.100.1.2.366426457713813178933224342280246227461",
-        "expectedFailures": ["GDCM", "Archetype"]
+        "expectedFailures": ["GDCM", "Archetype"],
+        "voxelValueQuantity": "(DCM, 110852, MR signal intensity)",
+        "voxelValueUnit": "(UCUM, [hnsf'U], Hounsfield unit)"
```

Comment:
I just wanted to avoid errors in the DICOM standard leak into Slicer. I highly value consistency but the DICOM standard is not consistent in itself, especially not among multiple domains (while Slicer covers all domains).

Anyway, if it's not that confusing and not terrible grammatically then I guess we can go with plural (units). In the documentation I'll describe why we chose plural and if it's not enough then we can add a singular version of the method name as well.